### PR TITLE
CPDP-1271: pick up test DMS details from config

### PR DESCRIPTION
**Pull request by mikail-khan**
Raising a separate PR for the app-config-* setup

### CODE REVIEW

File: app/uk/gov/hmrc/cgtpropertydisposals/service/DmsSubmissionService.scala [hunk 1]
```diff
@@ -56,6 +56,14 @@ class DefaultDmsSubmissionService @Inject() (
     extends DmsSubmissionService
     with Logging {
 
+  def getDmsMetaConfig[A: Configs](key: String): A =
+    configuration.underlying
+      .get[A](s"microservice.services.upscan-initiate.dms.$key")
+      .value
+
+  val queue: String        = getDmsMetaConfig[String]("classification-type")
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposals/service/DmsSubmissionService.scala]: are these values actually in application.conf? I can see entries for the path `dms.queue-name` and `dms.business-area`, but not under `microserice.services.upscan-initial.dms`

Comment (AndyHWChung) [reply app/uk/gov/hmrc/cgtpropertydisposals/service/DmsSubmissionService.scala]: should `.get[A](s"microservice.services.upscan-initiate.dms.$key")` be `.get[A](s"dms.$key")` and should this `classification-type`  be `queue-name`?

Comment (mikail-khan) [reply app/uk/gov/hmrc/cgtpropertydisposals/service/DmsSubmissionService.scala]: good spot! Yes they are but I changed the names actually. Let me fix that

Comment (mikail-khan) [reply app/uk/gov/hmrc/cgtpropertydisposals/service/DmsSubmissionService.scala]: fixed + fixed other config typos

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**